docs/developers/how-to-build-plugins.md

```diff
@@ -2,82 +2,111 @@
 sidebar-position: 1
 ---
 
-# How to build model plugins
+# How to build plugins
 
-The IF is designed to be as composable as possible. This means you can develop your own models and use them in a model pipeline.
-To help developers write Typescript models to integrate easily into IF, we provide the `ModelPluginInterface` interface. Here's an overview of the stages you need to follow to integrate your model:
+The IF is designed to be as composable as possible. This means you can develop your own plugins and use them in a pipeline.
+To help developers write Typescript plugins to integrate easily into IF, we provide the `PluginInterface` interface. Here's an overview of the stages you need to follow to integrate your plugin:
 
-- create a Typescript file that implements the `ModelPluginInterface`
-- install the model
-- initialize and invoke the model in your manifest file
+- create a Typescript file that manifestements the `PluginInterface`
+- install the plugin
+- initialize and invoke the plugin in your manifest file
 
-## The model interface
+## Step 1: Use our template repository
 
-The `ModelPluginInterface` is structured as follows:
+Instead of building up your plugin repository and all the configuration from scratch, you can use our [plugin template repository](https://github.com/Green-Software-Foundation/if-plugin-template). To use rthe template, visit the Github repository and click the `Use this template` button. You will have the option to `create a new repository` under your own account. Then, you can clone that repository to your local machine.
```

Comment:
@jmcook1186 typo : rthe instead of the

Reply:
```suggestion
Instead of building up your plugin repository and all the configuration from scratch, you can use our [plugin template repository](https://github.com/Green-Software-Foundation/if-plugin-template). To use the template, visit the Github repository and click the `Use this template` button. You will have the option to `create a new repository` under your own account. Then, you can clone that repository to your local machine.
```